pkg/controller/sharedcontrollerfactory.go

```diff
@@ -40,6 +51,8 @@ type sharedControllerFactory struct {
 	workers         int
 	kindRateLimiter map[schema.GroupVersionKind]workqueue.RateLimiter
 	kindWorkers     map[schema.GroupVersionKind]int
+
+	SyncOnlyChangedObjects bool
```

Comment:
The rest of these fields don't appear to be exported, is there some reason that this field needs to be? I'm not seeing any use of it outside this package.

Reply:
Fixed.